bin/build

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+function usage() {
+    echo -e "Usage: bin/build [TAG]
+
+OPTIONS:
+  -h, --help   print this message
+
+ARGUMENTS:
+  TAG        image tag in the format name:tag
+"
+}
+
+
+# Build moodlenet-backend image
+#
+# Usage: build TAG
+function build() {
+    tag="${1}"
+    version=$(cat docker/images/backend/UPSTREAM_VERSION)
+    docker_user="$(id -u):$(id -g)"
+
+    echo -e "${COLOR_INFO}Building the moodlenet-backend image with " \
+    "upstream version ${version}.${COLOR_RESET}"
+
+    docker build \
+        -f docker/images/backend/Dockerfile \
+        --build-arg BACKEND_VERSION="${version}" \
+        --build-arg DOCKER_USER="${docker_user}" \
+        --tag "${tag}" \
+        --target=moodlenet .
+
+    if [[ $(docker images -q "${tag}" | wc -l ) -ne 1 ]]; then
+        echo "Build failed! Image ${tag} is missing after completing the build"
+        exit 1
+    fi
```

Comment:
When (re)building with the same tag, the build may have failed and your test won't catch it. Shouldn't we consider that, if the build should fail, it would already have failed leading this script to exit prematurely?

Reply:
To be consistent, the `bin/build` script should rely on the `docker-compose build` command.

Reply:
That's right, the `set -e` should take care of this.
I removed these lines.